### PR TITLE
Makes bedsheets movable lights

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Bedsheets are now movable lights.

## Why It's Good For The Game

Less lag, faster updates.

## Testing Photographs and Procedure
image

## Changelog
:cl:
fix: Cosmic space bedsheet is now a movable light
/:cl:

### CODE REVIEW

**Review by itsmeow — DISMISSED** [review]
1 line change, amazing

File: code/game/objects/items/bedsheets.dm [hunk 1]
```diff
@@ -18,6 +18,7 @@
 	w_class = WEIGHT_CLASS_TINY
 	dying_key = DYE_REGISTRY_BEDSHEET
 	resistance_flags = FLAMMABLE
+	light_system = MOVABLE_LIGHT
```
Comment (imsynthex) [inline code/game/objects/items/bedsheets.dm]: This should not be applied to all bedsheets, it should instead only be added to `/obj/item/bedsheet/cosmos` and `/obj/item/bedsheet/double/cosmos` for the intended effect

Comment (PowerfulBacon) [reply code/game/objects/items/bedsheets.dm]: No other bedsheets have lights on them so it won't make an effect, if they somehow get lighting on them then they will use the movable light which is good

Comment (imsynthex) [reply code/game/objects/items/bedsheets.dm]: From testing this is not the case, every bedsheet has a small light effect by default
image

Comment (PowerfulBacon) [reply code/game/objects/items/bedsheets.dm]: Huh, didn't spot that. Why does the movable lighting system work like that...

**Review by imsynthex — APPROVED** [review]
Alright, looks good now

**Review by Penwin0 — APPROVED** [review]

**Pull request merged**